<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="07Clone Job Delete One"
    errors="1" failures="0" hostname="" id="" package="07Clone Job Delete One" skipped="0"
    tests="1" time="0.01300" timestamp="Tue, 24 Apr 2018 18:49:41 GMT">
  
    <testcase name="Create Two Jobs Delete One.js" classname="07Clone Job Delete One" time="0.01300" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [07 Clone Job Delete One / Create Two Jobs Delete One.js]:

  ReferenceError: dataProvider is not defined
    at Object.Create Two Jobs Delete One.js (/Users/Peizer/Documents/EventKitTest/debug/07Clone Job Delete One.js:33:74)
    at Module.call (/Users/Peizer/Documents/EventKitTest/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /Users/Peizer/Documents/EventKitTest/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/Users/Peizer/Documents/EventKitTest/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/Peizer/Documents/EventKitTest/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/Peizer/Documents/EventKitTest/node_modules/q/q.js:796:13)
    at /Users/Peizer/Documents/EventKitTest/node_modules/q/q.js:556:49
    at runSingle (/Users/Peizer/Documents/EventKitTest/node_modules/q/q.js:137:13)
    at flush (/Users/Peizer/Documents/EventKitTest/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    </system-err>
  

  
  </testsuite>
</testsuites>
